reana_workflow_controller/factory.py

```diff
@@ -12,13 +12,36 @@
 
 import logging
 
-from flask import Flask
+from flask import Flask, jsonify
+from marshmallow.exceptions import ValidationError
 from reana_commons.config import REANA_LOG_FORMAT, REANA_LOG_LEVEL
 from reana_db.database import Session
+from werkzeug.exceptions import UnprocessableEntity
+
 
 from reana_db.models import Base  # isort:skip  # noqa
 
 
+def handle_args_validation_error(error: UnprocessableEntity):
```

Comment:
This is the same error handler we use in `reana-server`. Should I move this to `reana-commons`?

Reply:
I am a bit biased here because I don't like `r-commons` and would suggest not to put it there. But another reason is that `webargs` versions differ between r-w-c and r-server. 

If you need someone else's opinion on this, feel free to tag them here.

Reply:
It mostly depends on whether we will continue using `webargs`, whether we will be able to update it to have the same version everywhere and if we will use it also in other components like r-job-controller. Maybe I can just put a `# FIXME` comment for now?